odm_sdk/scripts/upload_reference_genome.py

```diff
@@ -1,96 +0,0 @@
-#!/usr/bin/env python3
-# -*- coding: utf-8 -*-
-
-#
-# Copyright (c) 2011-2023 Genestack Limited
-# All Rights Reserved
-# THIS IS UNPUBLISHED PROPRIETARY SOURCE CODE OF GENESTACK LIMITED
-# The copyright notice above does not evidence any
-# actual or intended publication of such source code.
-#
-
-from odm_sdk import (DataImporter, FilesUtil,
-                              SpecialFolders, get_connection, make_connection_parser)
-
-
-def main():
-    """
-    Simple wrapper for uploading reference genome from public domain:  https://www.ensembl.org/index.html
-
-    For the details of FASTA file naming conventions, please see the README at:
-    ftp://ftp.ensembl.org/pub/release-91/fasta/homo_sapiens/dna/README
-    """
-
-    args = get_args()
-    connection = get_connection(args)
-    organism = args.species.replace("_", " ")
-
-    # Generate annotation_url if not specified
-    if args.annotation_url:
-        annotation_url = args.annotation_url
-    else:
-        annotation_url = f"ftp://ftp.ensembl.org/pub/release-{args.release}/gtf/{args.species.lower()}/" \
-                         f"{args.species}.{args.assembly}.{args.release}.gtf.gz"
-    # TODO: Use logger in future
-    print(f"Reference genome source URL:\n{annotation_url}")
-
-    # Generate name if not specified
-    if args.name:
-        name = args.name
-    else:
-        name = f"{organism} reference genome {args.assembly}.{args.release}"
-
-    # TODO: Use logger in future
-    print(f"Reference genome name:\n{name}")
-
-    # Create folder in GenestackFS
-    fu = FilesUtil(connection)
-    parent = fu.get_folder(fu.get_special_folder(SpecialFolders.CREATED),
-                           "Data samples", "Reference genome",
-                           create=True)
-
-    # Make api-call
-    accession = DataImporter(connection).create_reference_genome(
```

Comment:
data_importer.py should be also updated (create_reference_genome removed) + code in unified